backend/src/db/migrations/20200206190233-swap_latitude_with_longitude.js

```diff
@@ -0,0 +1,42 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description = `
+This migration swaps the value stored in Location.lat with the value
+of Location.lng. This is necessary as the values of lat and lng were
+stored incorrectly. For example Hamburg, Germany, was stored with the
+values lat=10.0 and lng=53.55, which is close to the horn of Africa,
+but it is lat=53.55 and lng=10.0
+`
+
+const swap = async function(next) {
+  const driver = getDriver()
+  const session = driver.session()
+  const transaction = session.beginTransaction()
+  try {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-14T11:33:08Z" title="Friday, February 14th 2020, 12:33:08 pm +01:00">Feb 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,37 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description =
+  'This migration swaps the value stored in Location.lat with the value of Location.lng.'
+
+const swap = async function(next) {
+  const driver = getDriver()
+  const session = driver.session()
+  const transaction = session.beginTransaction()
+  try {
+    // Implement your migration here.
+    await transaction.run(`
+        MATCH (l:Location) WHERE NOT(l.lat IS NULL)
+        WITH l.lng AS longitude, l.lat AS latitude, l AS Location
+        SET Location.lat = longitude, Location.lng = latitude
```


I prefer to use lower case for variables so as not to be confused with nodes... at first I thought you were trying to use set to set the node properties directly